relayer/Cargo.toml

```diff
@@ -19,6 +20,7 @@ sled = { version = "0.34.4", features = ["no_metrics", "no_logs"] }
 thiserror = "1.0.11"
 toml = "0.5"
 tracing = "0.1.13"
+tracing-futures = { version = "0.2.3" }
```

Comment:
It's not clear where was this necessary.

Reply:
no idea :) removed